easybuild/easyconfigs/m/MariaDB/MariaDB-10.0.21-GNU-4.9.3-2.25.eb

```diff
@@ -8,8 +8,9 @@ description = """MariaDB An enhanced, drop-in replacement for MySQL."""
 
 toolchain = {'name': 'GNU', 'version': '4.9.3-2.25'}
 
+source_urls = ['https://downloads.mariadb.org/interstitial/mariadb-%(version)s/source']
 sources = [SOURCELOWER_TAR_GZ]
-source_urls = ['https://downloads.mariadb.org/f/mariadb-10.0.21/source/']
+checksums = ['cad1471777c86f2be3050ed71360397c3999481cb8c00c317da077399eecf7ad']
```

Comment:
This checksum is different from what is was before (perhaps also for the others). Any idea what happened?

Reply:
Well, the new URL is actually wrong. Pushing the correct one now.